source/standards/programming-languages.html.md.erb

```diff
@@ -31,6 +31,16 @@ teams at the moment, and we are confident in how to host and operate
 applications written in them. We'd advise you to carry out new
 development in one of them.
 
+### Python
+
+New Python projects should be in Python 3.  Python 2 will
+[reach end of life in 2020][PEP373].  Python 3 is now well-supported
+by application frameworks and libraries, and is commonly used in
+production.  There is no longer any reason not to prefer Python 3 over
```

Comment:
Could we remove this final sentence? It's a double negative and I think what's there already does a good job at making the point.